exchange/src/App.tsx

```diff
@@ -83,7 +83,7 @@ function App() {
           {!id && (
             <Grid item xs={12} sm={12} md={5} lg={4}>
               <Typography color="primary" fontWeight={600} variant="h4">
-                <a href="https://www.cooperation.org">Cooperation.org</a>
+                <a href="https://repute.social/" target="_blank">Repute.social</a>
```

Comment:
Let's make the `S` of `Repute.Social` uppercase because that's how it is writtent in [landing page](https://repute.social)